src/components/commons/Pagination/index.tsx

```diff
@@ -0,0 +1,87 @@
+import Image from 'next/image';
+
+import classNames from 'classnames/bind';
+
+import { SVGS } from '@/constants';
+
+import usePagination from '@/hooks/usePagination';
+
+import styles from './Pagination.module.scss';
+
+const cx = classNames.bind(styles);
+const { left, right } = SVGS.pagination;
+
+type PaginationProps = {
+  totalCount: number;
+  onClick: (pageNumber: number) => void;
+};
+
+const Pagination = ({ totalCount, onClick }: PaginationProps) => {
+  const {
+    activePage,
+    currentPageGroupIndex,
+    currentPageGroup,
+    pagesArray,
+    handlePageClick,
+    handlePrevButtonClick,
+    handleNextButtonClick,
+    handleFirstPageClick,
+    handleLastPageClick,
+  } = usePagination(totalCount, onClick);
+
+  return (
+    <div className={cx('pagination')}>
```

Comment:
ul>li 사용해서 마크업하시는거 어떠신가요
```suggestion
<nav aria-label="Page navigation">
  <ul class="pagination">
    <li class="page-item"><button>1</button></li>
    <li class="page-item"><button>2</button></li>
    // ... 생략
  </ul>
</nav>
```

Reply:
반영했습니다!